src/test/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/BalanceSheetTransformerTests.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.web.accounts.transformer.smallfull;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class BalanceSheetTransformerTests {
+
+    private BalanceSheetTransformer transformer;
+
+    @BeforeEach
+    public void init() {
+
+        transformer = new BalanceSheetTransformer();
+    }
+
+    @Test
+    public void getBalanceSheetSuccess() {
```

Comment:
I believe you can get away with removing the access modifier on the tests now :)